components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/View.kt

```diff
@@ -68,6 +69,20 @@ fun View.hideKeyboard() {
     imm.hideSoftInputFromWindow(windowToken, 0)
 }
 
+/**
+ * Fills the given [Rect] with data about location view in the window.
+ *
+ * @see View.getLocationInWindow
+ */
+fun View.getRectWithViewLocation(out: Rect) {
```

Comment:
nit: have a default `Rect()` so that you can call this directly instead of using `Rect().apply`
```suggestion
fun View.getRectWithViewLocation(out: Rect = Rect()) {
```

Reply:
Fixed